test/builder-api/src/origins.js

```diff
@@ -111,6 +111,20 @@ describe("Origin API", function () {
     });
   });
 
+  describe("Create origin when not allowed", function () {
```

Comment:
Should we also have a test that proves the ability for allow listed users to be able to create an origin?